README.md

```diff
@@ -65,32 +65,36 @@ You can add `cwd=elmSource` to the loader:
 ```js
 var elmSource = __dirname + '/elm/path/in/project'
   ...
-  loader: 'elm-webpack?cwd=' + elmSource
+  use: 'elm-webpack-loader?cwd=' + elmSource
```

Comment:
it's better to use webpack 2 style options if everything is going to be converted.